examples/structs/structs.go

```diff
@@ -12,6 +12,16 @@ type person struct {
 	age  int
 }
 
+// A de facto constructor of type `person`.
```

Comment:
Not sure what "de facto" means here. Maybe something like:

    // NewPerson constructs a new person struct with the given name